src/cascadia/TerminalConnection/ConhostConnection.cpp

```diff
@@ -57,27 +62,41 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
 
     void ConhostConnection::Start()
     {
-        std::wstring cmdline = _commandline.c_str();
+        std::wstring cmdline{ _commandline.c_str() };

```

Comment:
I'd prefer `std::wstring cmdline{ _commandline.begin(), _commandline.end() };` because then we get guarantees about the bounds o the string we're using to initialize and we don't need to dip into the behind-the-scenes data members of `std::string`

Reply:
string has an explicit constructor from a `T` that has a data and size; maybe it can be used here?

Reply:
(`std::wstring cmdline{_commandline}`)